.github/workflows/sync.yml

```diff
@@ -45,7 +45,7 @@ jobs:
           # Upload all but TUF timestamp. Once timestamp is uploaded, all other files must uploaded.
           for f in $(ls repository/repository/ -I *timestamp.json)
           do
-            gcloud alpha --quiet storage cp --cache-control=no-store -r repository/repository/$f gs://sigstore-tuf-root/
+            gcloud alpha --quiet storage cp --cache-control=no-store -r repository/repository/$f gs://sigstore-preprod-tuf-root/
           done
           # Upload timestamp
-          gcloud alpha --quiet storage cp --cache-control=no-store -r repository/repository/*timestamp.json gs://sigstore-tuf-root/
+          gcloud alpha --quiet storage cp --cache-control=no-store -r repository/repository/*timestamp.json gs://sigstore-preprod-tuf-root/
```

Comment:
Should we also delete any files that are present in the preprod bucket but not in the repository/repository similar to sync to prod?

That way you can close out https://github.com/sigstore/root-signing/issues/243

Reply:
Yep, will add that!